Error: Cannot find module 'nodepie'

`ERROR Unable to load /home/hubot/node_modules/hubot-rss-alerts/src/rss-alerts: Error: Cannot find module 'nodepie'`

I installed following the directions. Keep getting this error no matter what.

--- a/package.json
+++ b/package.json
@@ -14,6 +14,7 @@
   "readmeFilename": "README.md",
   "main": "index.coffee",
   "dependencies": {
-    "hubot": ">= 2.4.0"
+    "hubot": ">= 2.4.0",
+    "nodepie": "^0.7.0"
   }
 }

